Docker/README.md

```diff
@@ -846,7 +887,7 @@ CONTAINER ID        IMAGE               COMMAND                  CREATED
 2573be6f1401        owncloud_web        "/bin/sh -c '/usr/sbi"   35 seconds ago      Up 34 seconds       0.0.0.0:80->80/tcp   owncloud_web_1
 ```
 
-Our application is thus started and should work the same way as previously. However it is much simpler as we don't need to precise ports and storage mapping, and this information can be managed in configuration through the YAML file.
+Our application starts and should work the same way as previously. However it is much simpler as we don't need to define ports and storage mapping, this information can be configured in the YAML file.
```

Comment:
For me managed in configuration also means using a Configuration Management System such as git (or VCS).